core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -800,6 +800,30 @@ object TestUtils extends Logging {
     leader
   }
 
+  def waitUntilLeadersAreMade(servers: Seq[KafkaServer], topic: String, partition: Int, timeout: Long = 5000L): Unit = {
```

Comment:
Hmm, this name can perhaps be improved a little, I think. Any suggestions @guozhangwang?